model/lora-net-device.cc

```diff
@@ -116,6 +121,7 @@ LoraNetDevice::Send (Ptr<Packet> packet)
   // Send the packet to the MAC layer, if it exists
   NS_ASSERT (m_mac != 0);
   m_mac->Send (packet);
+  m_promiscSnifferTrace (packet);
```

Comment:
We may think where we capture the packet. Maybe the MAC layer does not have duty cicle available (or whatever) and will not send this packet. We have to be the closest possible to the PHY layer (Lower-MAC probably) to capture the packet that effectively will go to the medium. See Discussion : https://github.com/signetlabdei/lorawan/pull/103